### PR TITLE
fix(chat):  Fix setting Overlay conversation, refactoring init selected conversations epic (Issue #2599, #2741)

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
**Description:**

- Refactoring init selected conversations epic
- Fix setting Overlay conversation (overlayConversationId option)

Issues:

- Issue #2599 
- Issue #2741 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2716.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2716.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2716.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/store/conversations/conversations.epics.ts [hunk 1]
```diff
@@ -234,90 +252,62 @@ const initSelectedConversationsEpic: AppEpic = (action$, state$) =>
...
+                  conversationIds: selectedConversationsIds,
                 }),
               ),
-              ...actions,
             );
           }
 
           return concat(
-            ...actions,
             of(
```
Comment (IlyaBondar) [inline apps/chat/src/store/conversations/conversations.epics.ts:316]: why did you remove `createNewConversations` action?

Comment (denys-kolomiitsev) [reply apps/chat/src/store/conversations/conversations.epics.ts:316]: It was not removed. This logic has been moved up to the line 258.

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/544788)

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2716.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — APPROVED** [review]
